app/Controllers/Partials/Story.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace App\Controllers\Partials;
+
+trait Story
+{
+
+    /*
+    Get an array of regions for a story. Returns false if there are no regions.
+    */
+    public function storyRegions ()
```

Comment:
This should be a static function as it's being called statically in the templates.